test/lti_1p3/tool/launch_validation_test.exs

```diff
@@ -33,6 +33,57 @@ defmodule Lti_1p3.Tool.LaunchValidationTest do
 
       assert {:ok, _lti_params} = LaunchValidation.validate(params, session_state)
     end
+
+    test "passes validation when aud claim is a list" do
+      jwk = jwk_fixture()
+      registration = registration_fixture(%{tool_jwk_id: jwk.id})
+      deployment_id = "1"
+
+      _deployment =
+        deployment_fixture(%{deployment_id: deployment_id, registration_id: registration.id})
+
+      state = "some-state"
+      session_state = state
+
+      claims =
+        all_default_claims()
+        |> put_in(["https://purl.imsglobal.org/spec/lti/claim/deployment_id"], deployment_id)
+        |> Map.put("aud", ["12345"])
+
+      id_token = generate_id_token(jwk, jwk.kid, claims)
+      params = %{"state" => state, "id_token" => id_token}
+
+      MockHTTPoison
+      |> expect(:get, fn _url -> mock_get_jwk_keys(jwk) end)
+
+      assert {:ok, _lti_params} = LaunchValidation.validate(params, session_state)
+    end
+
+    test "passes validation when JWK is not Base64URL encoded" do
+      jwk = jwk_fixture()
+      registration = registration_fixture(%{tool_jwk_id: jwk.id})
+      deployment_id = "1"
+
+      _deployment =
+        deployment_fixture(%{deployment_id: deployment_id, registration_id: registration.id})
+
+      state = "some-state"
+      session_state = state
```

Comment:
NIT: seems that until here is the same setup as the previos test so maybe we can group them, wdyt?

Reply:
Agree, if its possible to use an existing module instead of custom impl then we should